transparent-proxy/config/config.go

```diff
@@ -7,6 +7,8 @@ import (
 	"os/exec"
 )
 
+const DebugLogLevel uint16 = 7
```

Comment:
Should this be configurable?

Reply:
It is configurable, but that the default value we want to set. https://github.com/kumahq/kuma-net/pull/94/files#diff-7bfcad61fb7a1aff09a93d094f624c0ae9d1c2d8c1f419b3472032c45ed1e242R74